docs/Reference/Collection/CollectionManipulation.md

```diff
@@ -348,3 +373,32 @@ const collection = client.collection('some-collection');
 await collection.drop();
 // the collection "some-collection" no longer exists
 ```
+
+## collection.enableCollectionStream
+
+`async collection.enableCollectionStream(hasStreamBody): Object`
+
+Updates the collection stream flag.
+
+**Arguments**
+
+- **enableStream**: `boolean` 
+
+  Whether the stream should be enabled on the collection or not.
+
+  This parameter must be set to `true` when enabling a stream on the collection.
+
+**NOTE**
+    You can't set the flag to `false` as on-demand stream deletion is not allowed.
+
+**Examples**
+
+```js
+const client = new jsc8({url: "https://gdn1.macrometa.io", token: "XXXX"});
+//---- OR ----
+const client = new jsc8({url: "https://gdn1.macrometa.io", apiKey: "XXXX"});
+
+const collection = client.collection('some-collection');
+await collection.enableCollectionStream( { hasStream: enableStream } );
```

Comment:
update the document with latest changes for hasStream

Reply:
already updated the doc